api/src/wfl/api/routes.clj

```diff
@@ -39,7 +39,7 @@
             :swagger {:tags ["Informational"]}}}]
    ["/oauth2id"
     {:get {:summary   "Get the OAuth2 Client ID for this deployment of the server"
-           :handler   (handlers/success {:oauth2-client-id @once/oauth-client-id})
+           :handler   (fn [_] (handlers/succeed {:oauth2-client-id @once/oauth-client-id}))
```

Comment:
`succeed` is the same as `success`, not just using `constantly`, so it is properly lazy